src/pacmod3_node.cpp

```diff
@@ -204,6 +205,13 @@ void callback_turn_signal_set_cmd(const pacmod_msgs::SystemCmdInt::ConstPtr& msg
   lookup_and_encode(TurnSignalCmdMsg::CAN_ID, msg);
 }
 
+// Listens for incoming requests to change the state of the turn signals
+void callback_rear_pass_door_set_cmd(const pacmod_msgs::SystemCmdInt::ConstPtr& msg)
+{
+  lookup_and_encode(RearPassDoorCmdMsg::CAN_ID, msg);
+}
+
+
```

Comment:
Not a big deal, but there is an extra newline here. An auto-formatter missed this maybe?